geometry_msgs/Pose target_pose
---
string result